crates/esplora/src/blocking_ext.rs

```diff
@@ -50,6 +50,19 @@ pub trait EsploraExt {
     /// The full scan for each keychain stops after a gap of `stop_gap` script pubkeys with no associated
     /// transactions. `parallel_requests` specifies the max number of HTTP requests to make in
     /// parallel.
+    ///
+    /// ## Note
+    ///
+    /// `stop_gap` is defined as "the maximum number of consecutive unused addresses".
+    /// For example, with `stop_gap = 3`, `full_scan` will keep searching for script pubkeys
```

Comment:
```suggestion
    /// For example, with a `stop_gap` of  3, `full_scan` will keep scanning
```

Reply:
Done in b9ba9b5978c2dfb1ee4cd29ed565365bd62ee75f